modules/consensus/config.go

```diff
@@ -0,0 +1,29 @@
+package consensus
+
+import "gopkg.in/yaml.v3"
+
+// Config contains the configuration about chain start height
+type Config struct {
+	StartHeight int64 `yaml:"start_height,omitempty"`
+}
```

Comment:
It's not necessary to define a new structure for this thing honestly. We can simply get the config that is parsed when the `bdjuno` command is used and get the starting height from there: 

```go
// Get the config
startHeight := config.Cfg.ParserStartHeight
```

Reply:
Updated